README.md

```diff
@@ -51,8 +51,24 @@ section, `[all_backends]` was used: it is a shortcut to `[speculos,ledgercomm,le
 If the Speculos extra is installed (to use the `SpeculosBackend`), system dependencies are needed.
 [Check the doc](https://speculos.ledger.com/installation/build.html) for these.
 
-## Examples
+## Features
+
+The `src/ragger/backend/interface.py` file describes the methods that can be implemented by the different backends and that allow to interact with a device (either a real device or emulated) :
```

Comment:
In English (and most languages), there is not space before "double" ponctuation marks (`:`, `?`, `!`, ...). This is Frenchophone-specific.

Reply:
Ok